DigitalLearningSolutions.Web/Views/RegisterInternalAdmin/Confirmation.cshtml

```diff
@@ -1,18 +1,25 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.Register
-@model PersonalInformationViewModel
+@model AdminConfirmationViewModel
 
 @{
   ViewData["Title"] = "Admin registration complete";
+  var verifyEmailWarningViewModel = new ConfirmationVerifyEmailWarningViewModel(Model.UnverifiedPrimaryEmail, Model.UnverifiedCentreEmail, Model.CentreName);
 }
 
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
-    <h1 class="nhsuk-heading-xl">Admin registration complete</h1>
+    <h1 id="page-heading" class="nhsuk-heading-xl">Admin registration complete</h1>
 
     <p class="nhsuk-body-l">
       Your email address matched details stored with the centre, so your centre manager account has successfully been created.
     </p>
 
-    <vc:action-link asp-controller="Login" asp-action="ChooseACentre" link-text="Switch centre" />
+    <partial name="_VerifyEmailWarningForRegistrationConfirmationPage" model="@verifyEmailWarningViewModel" />

```

Comment:
Shouldn't assume that we'll have to show this on internal journeys.

Reply:
added a conditional